tests/core/test_training.py

```diff
@@ -122,31 +123,34 @@ async def test_training_script_with_restart_stories(tmp_path: Path):
     assert True
 
 
-def configs_for_random_seed_test() -> List[Text]:
-    # define the configs for the random_seed tests
-    return ["data/test_config/ted_random_seed.yaml"]
+async def test_random_seed(tmp_path: Path, monkeypatch: MonkeyPatch):
+    policies_config = {
+        "policies": [
+            {"name": TEDPolicy.__name__, "random_seed": 42},
+            {"name": RulePolicy.__name__},
+        ]
+    }
 
+    policy_train = Mock()
+    monkeypatch.setattr(TEDPolicy, "train", policy_train)
```

Comment:
mhm, if we mock this, then the test doesn't test anymore what it's supposed to test, no?

Reply:
fixed that